test/integration/062_defer_state_test/changed_models/table_model.sql

```diff
@@ -1,2 +1,3 @@
 {{ config(materialized='table') }}
 select * from {{ ref('ephemeral_model') }}
+-- {{ my_macro() }}
```

Comment:
Just want to double check that these should be sql commented but not jinja commented.

Reply:
Yup! I still want the macro to be picked up as a dependency, just without actually doing anything. I can make this clearer with a comment